### DISCUSSION TITLE
Alternative login

### DISCUSSION THREAD

**Discussion by staabm** [RecordReplay/devtools#4143] [General]
Privacy wise I try to not use google oauth login whenever possible.
google is doing way to much stuff with our/my personal data and I don‘t want to support google by providing them even more. The only provider even worse then google would be facebook.

I would love using a different oauth provider when loggin into replay, e.g. github login.
Do you have any plans regarding the login user story?

**Comment by jasonLaster** [comment]
Agreed that it would be nice to support GitHub as well. I think we'll be able to get to authentication early next year.

**Comment by jonbell-lot23** [comment]
March 15, 2022:

We added the ability to log in with an email a few months back. Might be worth adding GitHub since that's such a common approach with developer tools.

**Reply by jasonLaster** [reply]
To clarify - we added the ability to login with Okta a few months ago. General email login is still on the roadmap
